float64 timestamp
uint16 sym
uint8 pressed
string name
